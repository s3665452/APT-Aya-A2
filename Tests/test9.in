YBRLBURRLBYYLLLRUYLBRYLUYRLRRBLYURYLYBUYRYUBLYYURBYLYUBRYYLUYURULUUBRYLBUYBRLBUBLRBBLURRRBLUBLRUBUBU
Abby
Josh
turn 2 R 2
turn 4 L 3
turn 5 L 1
turn 3 Y 2
turn 0 B 3
turn 1 B 1
turn 0 U 4
turn 0 R 4
turn 0 Y 5
turn 0 L 5

turn 5 Y 2
turn 2 R 2
turn 0 Y 1
turn 4 L 5
turn 1 Y 5
turn 0 L 5
turn 0 U 3
tunr 3 L 1
turn 0 R 3
turn 0 Y 5
