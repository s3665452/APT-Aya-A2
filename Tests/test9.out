Factories:
0:  
1: 
2: 
3: 
4: 
5: 

Score for Abby: 0
Mosaic for Abby:
1:         . || . Y . . L
2:       . . || . . Y R .
3:     . . . || U . B . .
4:   . . U U || . . . . .
5: . . . . . || Y . . . .
broken: F
Score for Josh: 0
Mosaic for Josh:
1:         . || B . . . L
2:       . . || . . Y R .
3:     . . . || . L . . R
4:   . . R R || . . . . .
5: . . . . . || . . . L .
broken: 